website/integrations/services/opnsense/index.md

```diff
@@ -92,6 +92,10 @@ In OPNsense, go to _System/Settings/Administration_ and under _Authentication_ a
 
 ![](./opnsense2.png)
 
+### Step 7
+You can now either import user(s) or syncronise from Authentik LDAP. See https://docs.opnsense.org/manual/how-tos/user-ldap.html for more.
```

Comment:
Misspelling: "[synchronize](https://dictionary.cambridge.org/us/dictionary/english/synchronize)." Also, @tanberry, I don't think the optional pluralization of `user(s)` is grammatically or semantically useful; I recommend going with just "users".

Reply:
Agree, either works but let's go with "users"; it is more simple.